bower_components/katex/dist/images/Image-Licensing-and-Technical-Notes.txt

```diff
@@ -0,0 +1,70 @@
+Copyright and Licensing
```

Comment:
Maybe I'm late to the party here, but is it worth calling this file `README.txt`?